src/services/mock/json/GscanSubscriptionQuery.json

```diff
@@ -0,0 +1,27 @@
+{
+  "workflows": [
+    {
+      "id": "user|one",
+      "name": "one",
+      "status": "running",
+      "owner": "user",
+      "host": "localhost",
+      "port": 43079,
+      "stateTotals": {
+        "waiting": 1,
+        "expired": 0,
+        "preparing": 0,
+        "submit-failed": 0,
+        "submitted": 3,
+        "running": 1,
+        "failed": 0,
+        "succeeded": 0
+      },
+      "latestStateTasks": {
+        "running": [
+          "running-task"
+        ]
+      }
+    }
+  ]
+}
```

Comment:
To change with #543 , where I will have to return the JSON as an initial burst delta (as we are doing here for the tree data).